modules/db/src/main/scala/gem/dao/ObservationDao.scala

```diff
@@ -30,7 +40,7 @@ object ObservationDao {
        .unique
        .map { case (t, i) =>
          Observation(id, t, i, Nil)
-       }      
+       }
 
   def select(id: Observation.Id): ConnectionIO[Observation[Step[_]]] =
```

Comment:
It seems to me that this should return `ConnectionIO[Option[Observation[Step[_]]]]` much like the corresponding select in `ProgramDao`.  No?